### PR TITLE
More work on describe menu

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by dannyparsons** [comment]
You should be creating new dialogs in separate branches so they can be merged quickly, not on a branch for other work. I can't accept this until all the code is working well, and if someone else edits vbproj or frmMain before then this is finished, then this will conflict and you'll have to do the branch all over again.

File: instat/dlgOneVariableGraph.vb [hunk 1]
```diff
@@ -163,4 +163,8 @@ Public Class dlgOneVariableGraph
             rdoCombineGraph.Enabled = False
         End If
     End Sub
+
+    Private Sub ucrBase_BeforeClickOk(sender As Object, e As EventArgs) Handles ucrBase.BeforeClickOk
+        OneOrMoreVariables()
```
Comment (maxwellfundi) [inline instat/dlgOneVariableGraph.vb]: Ok, i will remove this part for now to make sure others do not get a conflict then fix it. It was working as desired though.

**Comment by maxwellfundi** [comment]
@dannyparsons  i have removed the before click okay sub, so you could merge the rest as  fix that

File: instat/dlgOneVariableGraph.vb (hunk 1 above)
Comment (maxwellfundi) [inline instat/dlgOneVariableGraph.vb]: Yeah, i think another option would be to run the OneorMoreVariables on NameChanged Event of the ucrinputcombo on the subdialog.

Comment (dannyparsons) [inline instat/dlgOneVariableGraph.vb]: It only seems to depend on the receiver so I don't think it is needed anywhere other than SelectionChanged. What is the problem? Maybe the sub itself is not correct.

Comment (dannyparsons) [inline instat/dlgOneVariableGraph.vb]: I need to merge this now so that it doesn't conflict. You can come back to this on a new branch.

**Pull request merged**